src/P.js

```diff
@@ -2,6 +2,9 @@ import React from 'react';
 import { styles, marg } from './styles';
 
 export default React.createClass({
+/**
+ * P is a typography component for rendering a paragraph with the proper styles.
+ */
```

Comment:
This is an open question, but P is the name of a component. I would consider potentially using using backticks around them: `P`.